section2/exercises/methods.rb

```diff
@@ -5,37 +5,56 @@
 
 # Example: Write a method that when called will print your name:
 def print_name
-  p "Severus Snape"
+  puts "Sam Cliffe"
 end
 
 print_name
 
 # Write a method that takes a name as an argument and prints it:
 def print_name(name)
-  # YOUR CODE HERE
+  puts (name)
 end
 
-print_name("Albus Dumbledore")
+print_name("Sam Cliffe")
 
-# Write a method that takes in 2 numbers as arguments and prints 
+# Write a method that takes in 2 numbers as arguments and prints
 # their sum. Then call your method three times with different arguments passed in:
-# YOUR CODE HERE
 
-# Write a method that takes in two strings as arguments and prints 
-# a concatenation of those two strings. Example: The arguments could be 
-# (man, woman) and the end result might output: "When Harry Met Sally".  
-# Then call your method three times with different arguments passed in. 
+def dog_total(dogs)
+  puts ("You have #{dogs} dogs!")
+end
+
+
+dog_total(4 + 1)
+dog_total(1 + 0)
+dog_total(99 + 5)
+
+
+# Write a method that takes in two strings as arguments and prints
+# a concatenation of those two strings. Example: The arguments could be
+# (man, woman) and the end result might output: "When Harry Met Sally".
+# Then call your method three times with different arguments passed in.
 
+# kind of a mix, a little shakey here
+def my_height(feet=6, inches=3)
+  puts ("You are " + feet.to_s + " feet, and " + inches.to_s + " inches tall")
+end
+
+my_height
+my_height(feet=5, inches=1)
```

Comment:
To supersede the default value, you don't need to resuse the variables

Example Update
```ruby
my_height(5, 1)
```
OR
```ruby
feet = 5
inches = 1
my_height(feet, inches)
```